docs/configuration/configuration.md

```diff
@@ -1545,6 +1549,95 @@ for a detailed example of configuring Prometheus for Kubernetes.
 You may wish to check out the 3rd party [Prometheus Operator](https://github.com/coreos/prometheus-operator),
 which automates the Prometheus setup on top of Kubernetes.
 
+### `<kuma_sd_config>`
+
+Kuma SD configurations allow retrieving scrape target from the [Kuma](https://kuma.io) control plane.
+
+This SD discovers "monitoring assignments" based on Kuma [Dataplane Proxies](https://kuma.io/docs/latest/documentation/dps-and-data-model),
+via the MADS (Monitoring Assignment Discovery Service) xDS API, and will create a target for each proxy
+inside a Prometheus-enabled mesh.
+
+Available xDS-specific meta labels:
+
+* `__meta_xds_server`: the xDS management server address, running on the Kuma Control Plane
+* `__meta_xds_protocol_version`: the xDS protocol version used to communicate with the server
+* `__meta_xds_client_id`: the xDS client ID sent to the server
+
+Available Kuma-specific meta labels:
+
+* `__meta_kuma_mesh`: the name of the proxy's Mesh 
+* `__meta_kuma_dataplane`: the name of the proxy
+* `__meta_kuma_service`: the name of the proxy's associated Service
+* `__meta_kuma_api_version`: the MADS API version used to communicate with ther server
+
+All user-defined [tags](https://kuma.io/docs/latest/documentation/dps-and-data-model/#tags) will be translated into
+labels, following the [Prometheus label spec](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels),
+under a common prefix that is guaranteed to not be overwritten by Prometheus in future versions.
+
+* `__meta_kuma_label_{tag}`:
```

Comment:
All the paragraph before can be replaced by `__meta_kuma_label_<labelname>: each label of the proxy`